web_view_transition/models/inherited_res_users.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    view_transition_mode = fields.Selection([
+        ('normal', 'Normal'),
+        ('scale-back-top', 'Scale (Back-Top)'),
+        ('scale-top-back', 'Scale (Top-Back)'),
+        ('skew', 'Skew'),
+        ('translate-left-right', 'Translate (Left-Right)'),
+        ('translate-right-left', 'Translate (Right-Left)'),
+        ('translate-top-down', 'Translate (Top-Down)'),
+        ('translate-down-top', 'Translate (Down-Top)'),
+        ('fade-in', 'Fade-In'),
+        ('circle-in', 'Circle-In'),
+        ('rotateX3D', 'Rotate X Top (3D)'),
+        ('rotateY3D', 'Rotate Y (3D)'),
+        ('rotateX2D', 'Rotate X'),
+    ], string="View Transition Mode", default='normal')
```

Comment:
The default of this value should be false / null, because you check if the field is 'populated' in the javascript file